alacritty/src/input.rs

```diff
@@ -828,7 +827,7 @@ impl<T: EventListener, A: ActionContext<T>> Processor<T, A> {
                 let delta_y = touch.location.y - last_touch.location.y;
                 *touch_purpose = TouchPurpose::Scroll(touch);
 
-                self.scroll_terminal(0., delta_y * TOUCH_SCROLL_FACTOR);
+                self.scroll_terminal(0., delta_y, 1.0);
```

Comment:
Could we have extra comment why it's done that way, so it won't get removed by accident?